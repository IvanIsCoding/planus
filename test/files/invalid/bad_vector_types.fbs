table Foo {
  x: [Bar];
  y: [[Foo: 1]];
}

union Bar {}