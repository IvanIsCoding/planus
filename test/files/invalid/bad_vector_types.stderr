[0m[1m[38;5;9merror[0m[1m: Unsupported type[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_vector_types.fbs:2:7
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: [[0m[31mBar[0m];
  [0m[34mâ”‚[0m       [0m[31m^^^[0m [0m[31mUnions in vectors are not currently supported[0m

[0m[1m[38;5;9merror[0m[1m: Unsupported type[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/bad_vector_types.fbs:3:7
  [0m[34mâ”‚[0m
[0m[34m3[0m [0m[34mâ”‚[0m   y: [[0m[31m[Foo: 1][0m];
  [0m[34mâ”‚[0m       [0m[31m^^^^^^^^[0m [0m[31mArrays in vectors are not currently supported[0m

Error: Bailing because of errors
